sdk/textanalytics/Azure.AI.TextAnalytics/src/ServiceClients/LegacyServiceClient.cs

```diff
@@ -1428,6 +1428,16 @@ public override Response<Models.AnalyzeTextJobStatusResult> AnalyzeStatusNextPag
             }
         }
 
+        public override Task CancelAnalyzeActionsJobAsync(string jobId, CancellationToken cancellationToken = default)
```

Comment:
Is it only .NET that has a Legacy and the new client? Does Java follow the same pattern?

Reply:
Yes, Java follows the same pattern but I don't know if they've done this yet. Python does, though.